ui/job-view/pushes/PushJobs.jsx

```diff
@@ -96,6 +109,23 @@ class PushJobs extends React.Component {
     jobInstance.toggleRunnableSelected();
   };
 
+  handleGroupClick = (ev, groupNode, groupInstance) => {
+    if (!groupInstance.state.expanded || !ev.metaKey) {
```

Comment:
clicking group name conflicts with `toggleExpanded`, so I added a special handling for `metaKey`,
there:
  * if metaKey is held, and the group is expanded, `toggleExpanded` does nothing and let `handleGroupClick` toggle selection
  * otherwise (including metaKey is held and the group is not expanded), perform normal `toggleExpanded`